environments/production/postgresql.yml

```diff
@@ -1,13 +1,13 @@
 postgres_override:
   allow_dump_from_pgstandby: yes
-  postgresql_max_connections: 300
+  postgresql_max_connections: 500
   postgresql_version: '9.6'
   postgresql_checkpoint_completion_target: '0.7'
   postgresql_wal_buffers: 16MB
   postgresql_max_wal_size: 2GB
 
 pgbouncer_override:
-  pgbouncer_default_pool: 290
+  pgbouncer_default_pool: 490
```

Comment:
Changing it here annoyingly changes it for _all_ production postgres dbs. I know how to override `postgresql_max_connections` for just a single instance, but I don't think we have a way to override the pgbouncer pool size for just a set of dbs?

Reply:
I think this is configurable in the inventory file on a per machine basis (per inventory group really). That would also prevent us from raising this beyond the postgres limit for any DBs

Reply:
Cal and I offlined a bit, but basically our code is set up to do this well if each postgresql machine also hosts pgbouncer—you just set `pgbouncer_default_pool` and `postgresql_max_connections` as inventory variables. The way we host in AWS envs, with a single pgbouncer machine pointing to multiple rds dbs, means that we would have to specify different values for different dbs in the same pgbouncer.ini config file. Since this isn't something we've tested, and we aren't even sure what the edge cases are there (you set `pool_size` per db, but if you have multiple dbs on a single host, do the pools sizes effectively get added together or is the largest one for a given host chosen?), so we opted to just raise the defaults and apply to all instances. No other db instances ever reach 300 connections currently, so raising that limit is in the short term largely theoretical anyway.